paddle/framework/op_desc.proto

```diff
@@ -51,6 +51,9 @@ message OpDesc {
     // type of this Operator, such as "add", "sub", "fc".
     required string type = 3;
 
```

Comment:
`device_type` may be more complex than a string, we can add this when need and think it     more careful